db/migrate/20150312082018_create_videos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVideos < ActiveRecord::Migration
+  def change
+    create_table :videos do |t|
+      t.string :video_id
+      t.string :amara_link
+      t.string :title
+      t.integer :priority
+      t.string :description
```

Comment:
This shouldn't be of type string. Longer fields such as descriptions should be of type text.